source/tutorial/model-hook.md

```diff
@@ -1,10 +1,10 @@
-Now, let's add a list of available rentals to the index template. We know that rentals will not be static, since eventually users will be able to add, update or delete them.  For this reason, we will need a _rentals_ model to save information about the rentals. However, to keep things simple at first, our model is just going to be a hard-coded array of JavaScript objects. Later, we'll switch to using Ember Data, a library for robustly managing model data.
+Now, let's add a list of available rentals to the index template. We know that rentals will not be static, since eventually users will be able to add, update, and delete them.  For this reason, we'll need a _rentals_ model to save information about the rentals. To keep things simple at first, we'll use a hard-coded array of JavaScript objects. Later, we'll switch to using Ember Data, a library for robustly managing data in our app.
 
 Here's what our homepage will look like when we're done:
 
 ![super rentals homepage with rentals list](../../images/models/super-rentals-index-with-list.png)
 
-Since the route handler is responsible for loading model data, that's where we'll start. Let's open `app/routes/index.js` and add our hard-coded data as the model:
+In Ember, our route handler is responsible for loading model data. Let's open `app/routes/index.js` and add our hard-coded data as the model:
```

Comment:
with the rephrasing I think "In Ember, route handlers are responsible for loading model data" is better suited